protostar-rust/src/lib.rs

```diff
@@ -41,16 +69,16 @@ fn internal_run_tests(input_path: &str) -> anyhow::Result<()> {
     Ok(())
 }
 
-#[pyfunction]
-fn run_tests(input_path: String) -> PyResult<()> {
-    internal_run_tests(&input_path.to_owned())
-        .map_err(|e| PyErr::new::<RuntimeError, _>(format!("{:?}", e)))?;
-
-    Ok(())
-}
-
-#[pymodule]
-fn rust_test_runner_bindings(_py: Python, m: &PyModule) -> PyResult<()> {
-    m.add_wrapped(wrap_pyfunction!(run_tests))?;
-    Ok(())
-}
+// #[pyfunction]
+// fn run_tests(input_path: String) -> PyResult<()> {
+//     internal_run_tests(&input_path.to_owned())
+//         .map_err(|e| PyErr::new::<RuntimeError, _>(format!("{:?}", e)))?;
+//
+//     Ok(())
+// }
+//
+// #[pymodule]
+// fn rust_test_runner_bindings(_py: Python, m: &PyModule) -> PyResult<()> {
+//     m.add_wrapped(wrap_pyfunction!(run_tests))?;
+//     Ok(())
+// }
```

Comment:
I think it is used in other PR

Reply:
Python bindings will get replaced with this CLI so this will stop being relevant. Having them defined alongside the code causes problems with build: maturin must be used instead of cargo, which is very much not ideal.

Reply:
I think we should remove them, no point in having them commented out like this.